.golangci.yml

```diff
@@ -1,6 +1,5 @@
 run:
   timeout: 10m
-  skip-dirs-use-default: true
```

Comment:
https://golangci-lint.run/usage/configuration/

The param should be `exclude-dirs-use-default` but it defaults to true.